### ISSUE TITLE
Val loss increase when training

### ISSUE THREAD

**Issue by Augustby** [li-ronghui/LODGE#20]
Thanks for your impressive work!
I'm trying to train LODGE model from scratch to reproduce the results. However, when training the global diffusion and finetuning the local diffusion, the train loss gradually decreased，while the val loss started to increase after a certain number of iterations. This suggests the model may be overfitting.
I followed the same config files as you provided [here](https://drive.google.com/file/d/13Yp__EPAw0EjrSS898X5FtSQGmveBykA/view?usp=sharing), but only modified the batch size to prevent OOM on my GPU.  Could you provide more detailed instruction on training to solve this problem and how to select the final checkpoints of two diffusion models?

Global Diffusion Loss
![global loss](https://github.com/li-ronghui/LODGE/assets/91830107/2b0bfe1e-889f-4be1-9ead-daf859fd48ce)
Local Diffusion Loss (finetune)
![local loss](https://github.com/li-ronghui/LODGE/assets/91830107/ab0a7b66-4451-4372-9829-e5583ef8adec)

**Comment by li-ronghui** [comment]
Currently, the Vloss/val in the code calculates the error between the velocity of generated data and the ground truth (GT). This make no practical sense in generation tasks, so please ignore it. Replacing Vloss with FID would make more sense, but this would slow down the training.
